### PR TITLE
MEN-6544: Integrate `mender-setup` and `mender-snapshot` repositories

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
I was not very sure on where to place them. I put them at the end of the 2x series: after `mender-connect` and `mender-configure-module` and before `mender-artifact` and `mender-cli` (3x series).

**Review by kacf — APPROVED** [review]
👍

**Review by tranchitella — APPROVED** [review]

File: 24.mender-snapshot/docs.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: mender-connect
```
Comment (kjaskiewiczz) [inline 24.mender-snapshot/docs.md:2]: shouldn't the title be `mender-snapshot`?

Comment (kjaskiewiczz) [inline 24.mender-snapshot/docs.md:2]: shouldn't the title be `mender-snapshot`?

Comment (kjaskiewiczz) [inline 24.mender-snapshot/docs.md:2]: shouldn't the title be `mender-snapshot`?

Comment (lluiscampos) [reply 24.mender-snapshot/docs.md:2]: Thank you

Comment (lluiscampos) [reply 24.mender-snapshot/docs.md:2]: https://github.com/mendersoftware/mender-docs-changelog/pull/59

**Pull request merged**